README.md

```diff
@@ -66,6 +69,27 @@ The supported metadata keys for omigami are (case insensitive):
 #### Notebooks
 You can find a [tutorial](https://github.com/omigami/omigami/blob/master/notebooks/spec2vec/tutorial.ipynb) notebook in the `/notebooks/` folder.
 
+### MS2DeepScore
+#### Quickstart
+
+```python
+from omigami import MS2DeepScore
+
+client = MS2DeepScore(token="your_token")
+
+mgf_file_path = "path_to_file.mgf"
+n_best_matches = 10
+include_metadata = ["Smiles", "Compound_name"]
+ion_mode = "positive"  # either positive or negative
```

Comment:
are we assuming already the ion_mode? I mean we can, and that's done for the future